src/utils/node.ts

```diff
@@ -24,7 +24,7 @@ import { Port } from '../types';
 import { getAPIClient } from '../client';
 import { DEFAULT_PORT_P2P, DEFAULT_PORT_RPC } from '../constants';
 
-const INSTALL_LISK_CORE_COMMAND = 'npm i -g lisk-core@^4.0.0-rc.0';
+const INSTALL_LISK_CORE_COMMAND = 'npm i -g lisk-core@^4.0.0-rc.1';
```

Comment:
you could have taken this as a flag input.. and default it to a specific release, but users can override.. otherwise for release we have keep updating this externally

Reply:
The `^` in the code ensures that the latest available version of Lisk Core v4 published to the NPM registry is installed. So, even if we do not update it in the future, it'll work as is.